edit/monitoring.coreos.com.receiver/types/opsgenie.vue

```diff
@@ -138,11 +143,11 @@ export default {
         <h3>Responders</h3>
         <ArrayList v-model="responders" :mode="mode" :default-add-value="defaultResponder" :show-header="true">
           <template v-slot:column-headers>
-            <div class="row mb-10">
+            <div class="row" :class="{'mb-15': isView}">
```

Comment:
Need to add a class that will add 75px margin right - otherwise the 'Send To' label is not aligned with the control below. 75px is the width used for the remove button in the rows below.